server/reflector/worker/process.py

```diff
@@ -317,6 +363,241 @@ async def process_multitrack_recording(
     )
 
 
+async def _discover_recording_tracks(room_name: str, recording_id: str) -> list[str]:
+    """Discover audio tracks for a recording by listing S3 objects.
+
+    Daily.co stores recordings in S3 with predictable paths:
+    - Pattern: {org_name}/{room_name}/{recording_ts}-{uuid}-cam-audio-{track_ts}.webm
+    - Example: monadical/daily-20251028130707/1761656985836-uuid-cam-audio-1761656986115.webm
+
+    Args:
+        room_name: Daily.co room name (e.g., "daily-20251028130707")
+        recording_id: Daily.co recording ID (not used in path, for logging)
+
+    Returns:
+        List of S3 keys for audio tracks, or empty list if none found
+    """
+    try:
+        storage = get_dailyco_storage()
+
+        # Daily.co S3 structure: {org_prefix}/{room_name}/
+        # The org prefix is typically the Daily.co domain prefix (e.g., "monadical")
+        # We'll try listing with just room_name as prefix, which should work
+        # if AWS_FOLDER is configured correctly in storage settings
+        prefix = f"{room_name}/"
+
+        all_keys = await storage.list_objects(prefix=prefix)
+
+        # Filter for audio tracks only
+        audio_tracks = [
+            key for key in all_keys if key.endswith(".webm") and "cam-audio" in key
+        ]
+
+        logger.debug(
+            "Discovered tracks for recording",
+            recording_id=recording_id,
+            room_name=room_name,
+            prefix=prefix,
+            total_objects=len(all_keys),
+            audio_tracks=len(audio_tracks),
+        )
+
+        return audio_tracks
+
+    except Exception as e:
+        logger.error(
+            "Failed to discover tracks for recording",
+            recording_id=recording_id,
+            room_name=room_name,
+            error=str(e),
+            exc_info=True,
+        )
+        return []
+
+
+@shared_task
+@asynctask
+async def poll_daily_recordings():
```

Comment:
I think it would be nice to be able to configure the "last 2 hours". Knowing the system worker fail with healthcheck for sometimes more than 24h...